README.md

```diff
@@ -136,34 +134,65 @@ There are many ways to contribute to Pa11y Webservice, we cover these in the [co
 If you're ready to contribute some code, follow the [setup guide](#setup). The project can be linted and unit tested immediately:
 
 ```sh
-make lint           # Lint the code
-make test-unit      # Run the unit tests
+npm run lint           # Lint the code
+npm run test:unit      # Run the unit tests
 ```
 
 The integration tests require the service to be running in the background, since they'll be checking its behaviour.
 
 1. Create a configuration file for the `test` mode; one can be created quickly with `cp config/test.sample.json config/test.json`
 1. Start the service in test mode with:
+
    ```sh
    NODE_ENV=test npm start &
    ```
 
    The `&` places the service into the background. An alternative approach is to run `NODE_ENV=test npm start`, suspend the process with `CTRL+z`, and finally run `bg` to place it into the background.
+
 1. ```sh
-   make test-integration    # Run the integration tests
-   make test                # Run both the integration tests and the unit tests mentioned above
+   npm run test:integration   # Run the integration tests
+   npm test                   # Run both the integration tests and the unit tests mentioned above
+   ```
+
+### Testing the GitHub Actions workflow `test.yml`
```

Comment:
```suggestion
### Locally testing the GitHub Actions workflow `test.yml`
```

I've got this right, haven't I? The aim here is to give people a way to test it locally? Reject my suggestion if not, please 😅

Reply:
That's clearer 🙂